bindings/java/src/org/sleuthkit/datamodel/OsAccountRealmManager.java

```diff
@@ -220,51 +220,134 @@ public OsAccountRealm createWindowsRealm(String accountSid, String realmName, Ho
 		return realm;
 	}
 	
+	
 	/**
-	 * Updates the specified realm in the database.
-     * NOTE: This will not merge two realms if the updated information exists
+	 * Get a windows realm by the account SID, or the domain name. The input SID
+	 * is an user/group account SID. The domain SID is extracted from this
+	 * incoming SID.
+	 *
+	 * If a realm is found but is missing either the addr or the realmName, then
+	 * the realm is updated.
```

Comment:
I'm not sure a reader will know about the equivalency of the account SID and the address when they read this.

Reply:
Changed it to"... missing either the SID or the realmName, ..."